upgrade_tests/drop_compact_storage_upgrade_test.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+import logging
+
+from dtest import Tester
+
+since = pytest.mark.since
+logger = logging.getLogger(__name__)
+
+VERSION_30 = 'github:apache/cassandra-3.0'
+VERSION_311 = 'github:apache/cassandra-3.11'
+
+
+@since('3.0', max_version='3.11')
+class TestDropCompactStorage(Tester):
```

Comment:
Should it be marked with `@pytest.mark.upgrade_test`?

Reply:
True! Thanks!